File: api/send.py
from flask import Flask, request, jsonify
import requests

app = Flask(__name__)

@app.route('/api/send', methods=['POST'])
def send_message():
    data = request.get_json()
    username = data.get('username')
    message = data.get('message')

    url = f'https://ngl.link/api/submit'
    payload = {
        'username': username,
        'question': message,
        'deviceId': 'some-device-id'
    }

    response = requests.post(url, data=payload)
    if response.status_code == 200:
        return jsonify({'success': True, 'message': 'Pesan berhasil dikirim.'})
    else:
        return jsonify({'success': False, 'message': 'Gagal mengirim pesan.'}), 500

File: api/send.js
function tampilkanNotifikasi(text) {
  const notif = document.getElementById('notifikasi');
  notif.innerText = text;
  notif.style.top = '30px';
  setTimeout(() => {
    notif.style.top = '-50px';
  }, 1600);
}

async function mulaiSpam() {
  const username = document.getElementById("username").value;
  const pesan = document.getElementById("pesan").value;
  const log = document.getElementById("log");

  if (!username || !pesan) {
    tampilkanNotifikasi("Isi semua kolom dulu!");
    return;
  }

  tampilkanNotifikasi("Spam dimulai");

  for (let i = 1; i <= 10; i++) {
    try {
      const res = await fetch(`https://ngl.link/api/submit`, {
        method: "POST",
        headers: {
          "Content-Type": "application/json",
        },
        body: JSON.stringify({
          username: username,
          question: pesan,
          deviceId: "xxxxxxxxxxxxxx"
        }),
      });

      const text = await res.text();

      if (text.includes("ok")) {
        log.innerHTML += `<div>Pesan ke ${i} berhasil dikirim by villain</div>`;
      } else {
        log.innerHTML += `<div style="color:red">Pesan ke ${i} gagal dikirim</div>`;
      }
    } catch (e) {
      log.innerHTML += `<div style="color:red">Error jaringan saat kirim pesan ke ${i}</div>`;
    }

    await new Promise(r => setTimeout(r, 500)); // Delay 0.5 detik
  }

  tampilkanNotifikasi("**Spam selesai**");
}
